contents/gpgpu/gpgpu.md

```diff
@@ -48,6 +48,18 @@ The SIMT (Single Instruction, Multiple Threads) architecture of GPUs means that
 - CUDA: [Official website](http://www.nvidia.com/object/cuda_home_new.html)
 - OpenCL: [Official website](https://www.khronos.org/opencl/)
 
+### § 1.6 GPGPU in action
+The benefits of GPGPU are even more pronounced when it comes to embedded systems and Internet of Things (IoT) applications, where computing power is often compromised for physical space, weight, and power consumption. NVIDIA's Jetson is a computation processor board that delivers the processing capability of the Tegra GPU unit in a compact and light package; the latest version - Jetson TX2, measures at only 50 mm x 87 mm - the size of a business card<sup>[[6]](#footnote6)</sup>.
+
+![NVIDIA's Jetson TX2](jetson_tx2.jpg "NVIDIA's Jetson TX2")
+
+Packed with a NVIDIA Pascal<sup>TM</sup> GPU with 256 CUDA cores at maximally 1300 MHz<sup>[[6]](#footnote6)</sup>, the TX2 is capable of parallel computing-intensive tasks such as real-time vision processing or deploying deep neural networks, allowing mobile platforms to overcome physical constraints and solve complex, real-world problems.
```

Comment:
<b>1.</b> On NVIDIA's website they always put the term on Pascal, as it's the name of their GPU architecture. I just removed it.
<b>2.</b> I wanted to say that the Pascal GPU is capable of parallel computing, but couldn't find a better evidence for it.

Reply:
2. I think reducing the number of jargon in the earlier section, and clarify others made it easier to understand this part. If you added a small part clarifying the significance of CUDA cores (`CUDA cores are parallel processors` or something along this line), it may help convey your point of the Pascal GPU being capable of parallel computing. Thoughts?